src/super_gradients/examples/deci_lab_export_example/deci_lab_export_example.py

```diff
@@ -21,19 +23,20 @@
 model.build_model(net)
```

Comment:
We need to make sure conversion works for our SG supported architectures (for example calling model.build_model("yolo_v5n")).
We need to at least do this for some architecture families in each task. You can take the code from our Quickstart notebooks for the different tasks and modify the number of epochs to something small, and add the deci lab upload callback to the phase callbacks as done here.